src/components/DefinitionEntry.js

```diff
@@ -91,6 +103,26 @@ export default class DefinitionEntry extends React.Component {
     return licenseExpression ? <span>{licenseExpression}</span> : <span>&nbsp;</span>
   }
 
+  getRevisionUrl(definition) {
+    const coordinate = get(definition, 'coordinates')
+    switch (coordinate.provider) {
+      case 'github':
+        return ''
+      case 'npmjs':
+        return `${this.getLocationURL(definition)}/v/${coordinate.revision}`
+    }
```

Comment:
need to fill in the other providers

Reply:
I attempted to add more providers. This was the best I could do with the data I had access to. I have urls for providers github, npmjs, nuget, and mavencentral.